commanddash/test/steps/workspace_query_test.dart

```diff
@@ -51,13 +51,11 @@ void main() {
             "type": "search_in_workspace",
             "query": "<422243666>",
             "workspace_object_type": "all",
-            "workspacePath":
-                "/Users/keval/Desktop/dev/welltested/projects/dart_files",
             "outputs": ["436621806"]
           },
           {
             "type": "prompt_query",
-            "query":
+            "prompt":
```

Comment:
@KevalPrajapati this and the prompt_query_step are still broken. please take a look.